src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationsInProgress.jsx

```diff
@@ -142,7 +144,7 @@ const ApplicationsInProgress = ({
                     formId={formId}
                     formTitle={formTitle}
                     lastSavedDate={lastSavedDate}
-                    presentableFormId={presentableFormId}
+                    {...(hasBenefit ? { presentableFormId } : {})}
```

Comment:
This works just fine but a nitpick.

I think it's simpler and more legible to just property assign
`presentableFormId={ hasBenefit ? presentableFormId : null }` or false

Reply:
I think the object spread from a ternary is more warranted when the objects are dynamically constructed. But when they're more literal, flatter assignments can just end up easier to debug later on.